rats-apps/test/python/rats_test/apps/test_annotated_container.py

```diff
@@ -0,0 +1,288 @@
+from collections.abc import Callable
```

Comment:
can you please explain the intent of each of the APIs you use here?
are they all for usage by general users?  in what scenarios?